### DISCUSSION TITLE
initialization with {...} expected for aggregate object?

### DISCUSSION THREAD

**Discussion by Skuttle-Studios** [nlohmann/json#3875] [Q&A] [answered]
I'm trying to initialize a JSON object with
`json j = ReadJSON("data.json");`
in Visual Studio. However, when I try to use the object with
`std::string e[5] = j.at("Types");`
it gives an error where I use the "j" variable that says "initialization with {...} expected for aggregate object."

How can I fix this?

**Comment by nlohmann** [comment, marked as answer]
`std::string e[5]` is an invalid declaration. Try

```cpp
auto e = j.at("Types").get<std::string>();
```

**Reply by Skuttle-Studios** [reply]
Thanks so much!
